### DISCUSSION TITLE
Doubts about the evaluation method

### DISCUSSION THREAD

**Discussion by ForestsKing** [PaddlePaddle/PaddleSpatial#19] [BaiduKDDCup2022] [answered]
It is assumed that the test set contains 7 days of data, and the length of each prediction is 2 days. According to the description in the report, my final score calculation method is: use the sliding window to predict 288 numbers each time, take the mean value of MAE and RMSE, and then sum the scores of all windows (expression 2 in the report). Why sum rather than average?

**Comment by zhoujingbo** [comment]
Hi, 

If we use the same number of sliding windows, we think the "sum" and "average" should have got the same rankings. 

Thanks
Regards
Jingbo

**Comment by ramber1836** [comment, marked as answer]
Hi

> It is assumed that the test set contains 7 days of data, and the length of each prediction is 2 days. According to the description in the report, my final score calculation method is: use the sliding window to predict 288 numbers each time, take the mean value of MAE and RMSE, and then sum the scores of all windows (expression 2 in the report). Why sum rather than average?

Hi there, 

Either 'sum' or 'average' is OK for evaluation in terms of ranking. 
To avoid confusion, we take the average score of all rolling windows as the evaluation metric. 
The report will be updated shortly to make sure consistency with the baseline code.
